packages/cli/lib/imageslice.js

```diff
@@ -171,49 +177,58 @@ export default async function () {
     // Verify expected images were sliced
     // Log success
     async function imagesDone() {
-      spinner.succeed(`Done processing ${imagesSliced} images.`);
       await compareSlicedResults();
       getAllImagesAndRename(path.normalize("static/img/iiif/processed"));
       resetConfigPath();
+      done();
     }
 
     // compared the finished folders with the injested results to see which failed/passed
     // subtract the amount failed from the iterations to display the correct amount
     // in imagesDone()
     async function compareSlicedResults() {
       const processedImages = fs.readdirSync(iiifProcessed);
-      let failed = [];
-      let completed = [];
       for (let i = 0; i < originalImages.length; i++) {
         const originalImageFile = path.parse(originalImages[i]).base;
         const originalImageFileName = path.parse(originalImages[i]).name;
         if (processedImages.includes(originalImageFileName)) {
-          completed.push(originalImageFile);
+          results.push({
+            image: originalImageFile,
+            status: 'success'
+          })
         } else {
-          failed.push(originalImageFile);
+          results.push({
+            image: originalImageFile,
+            status: 'error',
+            message: 'Ensure file has proper exif headings'
+          })
         }
       }
-      if (failed.length) {
-        spinner.fail(
-          `${failed.length}/${imagesSliced} failed. Ensure these files have proper exif headings: ${['', ...failed].join('\n - ')}`
-        )
-      }
-      if (completed.length) {
-        spinner.succeed(
-          `${completed.length}/${imagesSliced} succeeded:${['', ...completed].join('\n - ')}`
-        );
-      }
     }
 
     // done function, trigger finish and resolve true to the Promise
     function done() {
-      spinner.succeed("IIIF Process Finished!");
+      if (results.length) {
+        const failures = results.filter((image) => image.status === "error");
+        spinner.clear();
+        console.log(`::Summary::`);
+        console.log(`Processed ${results.length} ${results.length === 1 ? 'image' : 'images'}${failures.length ? ` with ${failures.length} ${failures.length === 1 ? 'failure' : 'failures'}` : ''}.`);
+        if (failures.length) {
+          console.log('\n');
+          console.log('Unable to process the following files.')
+          failures.forEach((result) => {
+            spinner.fail(result.image+' - '+result.message);
```

Comment:
I am not certain if readability is easier or equivalent using a string template literal, thoughts?
```suggestion
            spinner.fail(`${result.image} - ${result.message}`);
```